src/ert/scheduler/lsf_driver.py

```diff
@@ -292,13 +292,15 @@ async def submit(
         executable: str,
         /,
         *args: str,
-        name: str = "dummy",
+        name: str | None = None,
         runpath: Path | None = None,
         num_cpu: int | None = 1,
         realization_memory: int | None = 0,
     ) -> None:
         if runpath is None:
             runpath = Path.cwd()
+        if name is None:
+            name = Path(executable).name
```

Comment:
This spurred this "bug": https://github.com/equinor/ert/issues/9564 (but it does not have to be solved in this PR since the problem existed beforehand)